src/web/modules/gateway_accounts/detail.njk

```diff
@@ -218,6 +214,33 @@
     </div>
   {% endif %}
 
+  {% if account.block_prepaid_cards === true %}
+    <div>
```

Comment:
Potentially to reduce some duplication this could use the nunjucks `set` syntax, something like: 
```html
{% set prepaidBlockTitle = "Block prepaid cards" if account.block_prepaid_cards else "Allow prepaid cards" %}
{% set prepaidBlockDescription %}
  {% if account.block_prepaid_cards %}
  <p class="govuk-body">...</p>
  <p class="govuk-body">...</p>
  {% else %}
  <p class="govuk-body">...</p>
  <p class="govuk-body">...</p>
  {% endif %}
{% endset %}

<h1 class="govuk-heading-s">{{ prepaidBlockTitle }}</h1>

{{ prepaidBlockDescription }}

{{ govukButton({
      text: prepaidBlockTitle,
      href: "/gateway_accounts/" + gatewayAccountId + "/block_prepaid_cards/toggle"
      })
    }}
```